a3p-integration/proposals/b:upgrade-14/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "agoricProposal": {
+    "releaseNotes": "TBD",
+    "sdkImageTag": "unreleased",
+    "planName": "UNRELEASED_UPGRADE",
+    "upgradeInfo": {
+      "coreProposals": [
+        "@agoric/builders/scripts/vats/init-network.js"
+      ]
+    },
+    "type": "Software Upgrade Proposal"
+  },
```

Comment:
Why is this a 2nd chain software upgrade? it should be in the existing `a:upgrade-next` on master.

Reply:
done